Source: Tom Briden <tom@decompile.me.uk>
Upstream: No
Reason: The include of <GL/glext.h> from angle pulls in the system
file as provided by mesa that defines GL_KHR_robustness without
defining GL_CONTEXT_LOST_KHR resulting in a build error later on


diff --git a/ui/gl/gl_bindings.h.old b/ui/gl/gl_bindings.h
index adab4ab..70f8151 100644
--- a/ui/gl/gl_bindings.h.old
+++ b/ui/gl/gl_bindings.h
@@ -360,9 +360,12 @@
 #define GL_UNKNOWN_CONTEXT_RESET_KHR      0x8255
 #define GL_RESET_NOTIFICATION_STRATEGY_KHR 0x8256
 #define GL_NO_RESET_NOTIFICATION_KHR      0x8261
-#define GL_CONTEXT_LOST_KHR               0x0507
 #endif /* GL_KHR_robustness */
 
+#ifndef GL_CONTEXT_LOST_KHR
+#define GL_CONTEXT_LOST_KHR               0x0507
+#endif /* GL_CONTEXT_LOST_KHR */
+
 #ifndef GL_EXT_texture_norm16
 #define GL_EXT_texture_norm16 1
 #define GL_R16_EXT 0x822A
